code/modules/projectiles/ammunition/magazines.dm

```diff
@@ -286,4 +286,26 @@ obj/item/ammo_box/magazine/tommygunm45
 /obj/item/ammo_box/magazine/m12g/stun
 	name = "shotgun magazine (12g taser slugs)"
 	icon_state = "m12gs"
-	ammo_type = /obj/item/ammo_casing/shotgun/stunslug
\ No newline at end of file
+	ammo_type = /obj/item/ammo_casing/shotgun/stunslug
+//l85 ammunition
+/obj/item/ammo_box/magazine/l85
+	name = "L85A2 5.56x45mm magazine"
+	icon = 'icons/obj/buttlauncher.dmi'
+	icon_state = "full"
+	origin_tech = "combat=3"
+	ammo_type = /obj/item/ammo_casing/a556
+	caliber = "556"
+	max_ammo = 30

```

Comment:
30 ammo, 30 ammo for a craftable gun which will easily be obtainable by science and no one else.